Jenkinsfile

```diff
@@ -264,6 +264,16 @@ def runBuild(buildFlags, instrumentationTestTarget) {
         step([$class: 'LintPublisher'])
       }
     },
+    'JVM8 Examples check' : {
+      try {
+        sh "JAVA_HOME=\$JAVA8_HOME ./gradlew checkExamples ${buildFlags} --stacktrace"
```

Comment:
This seems pretty heavy-handed? Do we really need to check all samples for Java 8 compatibility?

Reply:
Also, does this really compile all samples? I noticed in the kotlin repo I needed to clean the build as otherwise Gradle thought the samples where already compiled (which they where, but with Java 11)